instana/span.py

```diff
@@ -450,6 +459,12 @@ def _populate_exit_span_data(self, span):
             self.data["gcs"]["projectId"] = span.tags.pop('gcs.projectId', None)
             self.data["gcs"]["accessId"] = span.tags.pop('gcs.accessId', None)
 
+        elif span.operation_name == "gcps-producer":
+            self.data["gcps"]["op"] = span.tags.pop('gcps.op', None)
+            self.data["gcps"]["projid"] = span.tags.pop('gcps.projid', None)
+            self.data["gcps"]["top"] = span.tags.pop('gcps.top', None)
+            self.data["gcps"]["sub"] = span.tags.pop('gcps.sub', None)
```

Comment:
Similar to the above, producers never send subscriptions.

Reply:
ACK